spec/experiments/string_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe String do
+  describe '#<<' do
+    example "文字の追加" do
```

Comment:
```suggestion
    example '文字の追加' do
```
https://github.com/bake0937/baukis2/pull/1/files#discussion_r471468329 と同様です